app/controllers/api/assignments_controller.rb

```diff
@@ -5,50 +5,56 @@ class Api::AssignmentsController < ApplicationController
 
     def create
         authorize Assignment
-
-        created_assignments = []
-        created_action_items = []
-        participant_ids = bulk_assignment_params.fetch(:participant_ids, [])
-        action_items = bulk_assignment_params.fetch(:assignments, [])
-
-        if action_items.empty? || participant_ids.empty?
-            render json: { error: 'Action items and Participants must be populated'}, status: :unprocessable_entity
+        participant_ids_string = single_assignment_params.fetch(:participant_ids)
```

Comment:
@didvi the change from single assignment to multiple assignment is going to change behavior on the frontend. What should happen if one of the middle API requests fails? What should the modal look like in this case?